ios/browser/url_sanitizer/BUILD.gn

```diff
@@ -0,0 +1,19 @@
+source_set("url_sanitizer") {
```

Comment:
I assume brave_privacy stuff should be here?

Reply:
no more brave privacy stuff.